src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -147,7 +152,14 @@ private void transferEnchantments(ItemStack item, ItemStack book, Map<Enchantmen
     private boolean isDisenchantable(@Nullable ItemStack item) {
         if (item == null) {
             return false;
-        } else if (item.getType() != Material.BOOK) {
+        } else if (item.hasItemMeta() && item.getItemMeta().hasLore()) {
+            for (String lore : cantDisenchantLores.getValue()) {
+                if (item.getItemMeta().getLore().contains(ChatColors.color(lore))) {
```

Comment:
You are calling getItemMeta() twice, assign it to a variable before you check hasLore()

Reply:
edited